Added ability to clean a single address

RomainVialard opened the pull request:
Useful when you need to create an email address out of a name

RomainVialard committed:
Added ability to clean a single address

Useful when you need to create an email address out of a name

JeanRemiDelteil commented on src/email_validator.gs.js:
```diff
@@ -129,6 +130,19 @@ function cleanUpEmailList(emails, options) {
+ * @return {string} cleaned-up email address ("Hervé.Du Chène@gmail.com" --> "Example Foo") or null
+ */
+function cleanUpSingleAddress(email) {
+  return cleanUpEmailList(email.replace(/\s/g, ''), {
```
I would move the space removing code to the main clean up function, so the whole cleanup process is consistent ?

RomainVialard replied:
Done

RomainVialard committed:
moved space removing code

moved the space removing code to the main clean up function, so the whole cleanup process is consistent
